pkg/operator/ceph/pool/peers.go

```diff
@@ -0,0 +1,43 @@
+package pool
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/daemon/ceph/client"
+	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileCephBlockPool) reconcileAddBoostrapPeer(pool *cephv1.CephBlockPool,
```

Comment:
Please move this to the controller package and export it so it can be consumed in both rbd and pool places. Thanks.

Reply:
RBD implementation is a bit different because it also checks for the pool mirroring info [here](https://github.com/rook/rook/blob/baaea4a1ea0578dd5a9b25694e7d6cfd7c9cc18a/pkg/operator/ceph/cluster/rbd/config.go#L125).  Also we might remove peer import in RBD as we are deprecating it.  Do we still want to use same method for both rdb and pool ?

Reply:
@leseb Should it go in breaking changes section?

Reply:
Not yet, only when we remove it.